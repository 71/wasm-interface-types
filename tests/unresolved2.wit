;; parse-fail

(module
  (@interface export "x" (func $foo)))

