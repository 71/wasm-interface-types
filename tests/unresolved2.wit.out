failed to find adapter func named `$foo`
     --> ../../tests/unresolved2.wit:4:32
      |
    4 |   (@interface export "x" (func $foo)))
      |                                ^
